js/app/src/Blocks.svelte

```diff
@@ -525,22 +525,7 @@
 				.on("status", ({ fn_index, ...status }) => {
 					tick().then(() => {
 						const outputs = dependencies[fn_index].outputs;
-						outputs.forEach((id) => {
```

Comment:
We can also delete the line above (508) which defines `const pending_outputs: number[] = [];`

Reply:
We also have some additional logic involving `outputs_set_to_non_interactive` that can be deleted

Reply:
Ah yeh.